test/bundle_update_interactive/cli/row_test.rb

```diff
@@ -0,0 +1,65 @@
+# frozen_string_literal: true
+
+require "test_helper"
+
+class BundleUpdateInteractive::CLI
+  class RowTest < Minitest::Test
+    def test_formatted_gem_name_for_vulnerable_gem_is_red_on_white
+      outdated_gem = build(:outdated_gem, name: "rails", vulnerable: true)
```

Comment:
🗒️ These tests were super easy to write, because I can now use FactoryBot's `build` to construct the exact scenarios I needed.